docs/setup/nao_image_and_sdk.md

```diff
@@ -37,178 +37,136 @@ mkdir yocto/
 All these things are updated in the regular Yocto releases.
-To ensure deterministic builds the HULKs freeze versions of all used layers in the `kas-project.yml` files of meta-nao and meta-hulks.
+To ensure deterministic builds the HULKs freeze versions of all used layers in the `kas-project.yml` files of meta-nao.
 
-### Advanced: Upgrade Image/SDK Versions and Semantic Versioning
+#### Upgrade Image/SDK Versions and Semantic Versioning
 
 The HULKs use semantic versioning for the Yocto images and SDKs.
 This means that versions are increased depending on the severity of changes.
 The following policy exists for the HULKs:
 
--   Both images and SDKs have major, minor, and patch version numbers (e.g. 4.2.3)
--   Images and SDKs with the same major and minor version number are compatible with each other
--   Major changes, refactorings, implementations result in the increase of the major version number
--   Minor changes, additions and iterations result in the increase of the minor version number
--   Changes in the image that do not require SDK recreation, result in the increase of the patch version number (which only requires to create a new image)
+-   Both images and SDKs have major, minor, and patch version numbers (e.g. 4.2.3).
+-   Images and SDKs with the same major and minor version number are compatible with each other.
+-   Major changes, refactorings, or implementations result in the increase of the major version number.
+-   Minor changes, additionsk, and iterations result in the increase of the minor version number.
+-   Changes in the image that do not require SDK recreation, result in the increase of the patch version number. This consequently only requires to create a new image and not necessarily a redistribution of new SDKs.
 
-Before building new images, the version number needs to be set in `meta-hulks/conf/distro/HULKsOS.conf`.
+Before building new images, the version number needs to be set in `meta-nao/conf/distro/HULKsOS.conf`.
 Only change the `DISTRO_VERSION`, the `SDK_VERSION` is automatically derived from the `DISTRO_VERSION`.
 
 Once a new image and/or SDK is released, pepsi needs to know the new version numbers.
 Therefore update the variables `OS_VERSION` and/or `SDK_VERSION` in `crates/constants/src/lib.rs`.
 Successive builds with pepsi will use the new version.
 
-### Advanced: Upgrade Rust Version
+#### Upgrade Rust Version
 
 Since upgrading the Rust version often requires manual steps, this section describes the approach on how to upgrade and generate the needed patch files.
 These instructions can be followed e.g. if a new Rust version is available and a new image/SDK should be created with this new version.
 Users that just want to use the current version that we upgraded to should skip this section.
-The latest patch set for is included in the meta-hulks layer (in `patches/`) or HULKs code release (in `yocto/meta-hulks/patches/`).
+The latest patch set for is included in the meta-nao layer (in `patches/`).
```

Comment:
I don't think we currently have such a patch.
[Link](https://github.com/HULKs/meta-nao/tree/main/patches).

Reply:
done